### PR TITLE
RAINCATCH-778 search for worker

### PR DESCRIPTION
**Pull request by witmicko**
#What
User search was broken

#How
added refresh dataset function and running filter on workers returned by `list` rather than ones in the scope.

#JIRA
https://issues.jboss.org/browse/RAINCATCH-778

### CODE REVIEW

**Comment by wtrocki** [comment]
[Reviewed]

**Comment by wtrocki** [comment]
[Reviewed]

File: lib/worker-list/worker-list-controller.js [hunk 1]
```diff
@@ -13,11 +13,16 @@ var CONSTANTS = require('../constants');
 function WorkerListController($scope, mediator, userMediatorService, $timeout) {
   var self = this;
 
-  userMediatorService.listUsers().then(function(workers) {
-    $timeout(function() {
-      self.workers = workers;
+  function refreshWorkers() {
+    userMediatorService.listUsers().then(function(workers) {
+      $timeout(function() {
+        _workers = workers;
```
Comment (wtrocki) [inline lib/worker-list/worker-list-controller.js:20]: Variable is not defined. Javascript will manage here but the scope of the variable will be different so _workers will be undefined in line: 39.

See files implementation: https://github.com/feedhenry-raincatcher/raincatcher-file-angular/blob/a76d406ca62d6c29da3caf94cc9db75f343ec797/lib/file-list/file-list-controller.js#L11-L22

**Review by wtrocki — APPROVED** [review]

File: lib/worker-list/worker-list-controller.js (hunk 1 above)
Comment (witmicko) [reply lib/worker-list/worker-list-controller.js:20]: fixed, thank you.

**Pull request merged**